src/libs/actions/connections/index.ts

```diff
@@ -255,14 +257,20 @@ function syncConnection(policyID: string, connectionName: PolicyConnectionName |
         },
     ];
 
-    const parameters: SyncPolicyToQuickbooksDesktopParams = {
+    let parameters: SyncPolicyToQuickbooksDesktopParams | SyncPolicyToNSQSParams = {
```

Comment:
```suggestion
    const parameters: SyncPolicyToQuickbooksDesktopParams | SyncPolicyToNSQSParams = {
```
We can keep this as a const and avoid reassigning